ogre2/src/Ogre2ParticleNoiseListener.cc

```diff
@@ -88,6 +89,7 @@ void Ogre2ParticleNoiseListener::preRenderTargetUpdate(
 
       // get particle scatter ratio value from particle emitter user data
       // and pass that to the shaders
+      float scatterRatio = 0.1f;
```

Comment:
the default value before the change was 0.65 (in ogre2/src/Ogre2ParticleNoiseListener.hh) so I think we should keep that consistent

Reply:
Fixed in https://github.com/ignitionrobotics/ign-rendering/pull/275/commits/531ab80c95bd8cf0c85982453f7fc0c64a1ab56d